packages/qwik/src/core/shared/utils/jsx-filename.ts

```diff
@@ -0,0 +1,12 @@
+import type { DevJSX } from '../jsx/types/jsx-node';
+
+export function getFileNameFromJsx(jsxDev?: DevJSX): string | null {
```

Comment:
Can you rename filename to fileLoc or something similar so it's clear the position is added?